personalwebsite/templates/pagination.html

```diff
@@ -0,0 +1,19 @@
+<nav>
+    <ul class="pagination">
+        <li class="page-item">
+            {% if page.has_previous %}
+                <a class="page-link" href="?page={{page.previous_page_number}}">Previous</a>
+            {% endif %}
+        </li>
+
+        <li class="page-item">
+            <a class="page-link">Page {{page.number}} of {{page.paginator.num_pages}}</a>
```

Comment:
Really good! I like that you made this reusable so that any other html file that needs this UI,can quickly include it.
Only request I have is the same as the first review, add a single space at the start and end of the template tags.